crates/swift-bridge-ir/src/codegen/generate_swift.rs

```diff
@@ -73,18 +73,18 @@ impl SwiftBridgeModule {
                     continue;
                 }
             }
-
             let func_definition = match function.host_lang {
                 HostLang::Rust => {
-                    gen_func_swift_calls_rust(function, &self.types, &self.swift_bridge_path)
+                    let result =
```

Comment:
Calling this `result` makes it seem like we return a `Result`. Can we change this back?